packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -66,8 +69,8 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
                     className={get(this.props, "managedClasses.checkbox_input")}
                     type="checkbox"
                     ref={this.inputRef}
-                    onChange={this.props.onChange}
-                    checked={this.props.checked}
```

Comment:
why are you removing the `checked` property in the render method? It seems now that the `checked` prop never gets used except to set `defaultChecked` - is that intentional?

Reply:
Originally I was just coming in to update the styling cross-browser, but then noticed this page errored due to controlled/uncontrolled state handling. I've re-wired it again, I believe it handles both states now.

Reply:
I still don't understand why you removed the `checked` prop though - this means that the checkbox cannot be manually assigned a value

Reply:
It's still there. The `defaultChecked` is for the controlled value, while the `checked` is for the uncontrolled.

Reply:
Ok - made some changes. Should be good.